.github/workflows/test-action.yaml

```diff
@@ -38,10 +38,11 @@ jobs:
             ${{ github.workspace }}/action/node_modules
           key: ${{ runner.os }}-node-16-action-${{ hashFiles('action/yarn.lock') }}
       - name: Install dependencies
-        working-directory: .
+        working-directory: action
```

Comment:
This action has a default `working-directory` set on line 22. This line can be removed.